python/mooseutils/PerfGraphReporterReader.py

```diff
@@ -9,6 +9,9 @@
 
 from mooseutils.ReporterReader import ReporterReader
 
+# Use old sum() behavior (Python <3.12)
```

Comment:
Is all of the r""" in here a replacement failure?

Reply:
Python 3.12 required it for unittests. If I missed one, I'll add it.

Reply:
Oh I understand the question.

r""" is required when there are regular expressions within the docstring. Why 3.12 complains about this... 🤷‍♂️ 

That is why some have them, while others do not.

Reply:
https://discuss.python.org/t/getting-syntax-errors-in-docstrings-how-to-fix/50104